requirements-rocm.txt

```diff
@@ -8,4 +8,5 @@ botocore
 ray >= 2.10.0
 peft
 pytest-asyncio
-tensorizer>=2.9.0
\ No newline at end of file
+tensorizer>=2.9.0
+torch >= 2.4.0
```

Comment:
This shouldn't be specified here in its current form because it will go straight to trying to install the CUDA version of PyTorch.

If you're already building Torch then this becomes redundant. If you're not building Torch then you need to install recent enough wheels (**2.5.0**) and need to supply the ROCm-specific index urls.